tests/jobs/scala/src/main/scala/HDFSWordCount.scala

```diff
@@ -0,0 +1,11 @@
+import org.apache.spark.{SparkConf, SparkContext}
+
+object HDFSWordCount {
+  def main(args: Array[String]): Unit = {
+    val conf = new SparkConf()
+    val sc = new SparkContext(conf)
+    val file = args(0)
+    val rdd = sc.textFile(file)
+    print(s"number of words in ${file}: ${rdd.count()}")
```

Comment:
Since you use an input file from HDFS (`hdfs://<file>`) to exercise the communication, why not also write back to HDFS? E.g. `rdd.saveAsTextFile("hdfs://result.txt")`